kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/LinuxNetworkUtil.java

```diff
@@ -65,13 +71,12 @@ public class LinuxNetworkUtil {
     private static final String IFCONFIG = "ifconfig";
     private static final String IWCONFIG = "iwconfig";
     private static final String IP = "ip";
-
     private static final String LINE_MSG = "line: {}";
-
     private static final String ERR_EXECUTING_CMD_MSG = "error executing command --- {} --- exit value={}";
-
     private static final String FAKE_MAC_ADDRESS = "12:34:56:78:ab:cd";
-
+    private static final String ETHTOOL_COMMAND = "ethtool";
+    private static final String[] DEFAULT_PATH = new String[] { "/sbin/", "/usr/sbin/", "/bin/", "/usr/bin/" };
```

Comment:
This is probably out of scope therefore feel free to ignore this comment.

Wouldn't it be better to retrieve these paths with:

```suggestion
    private static final String[] DEFAULT_PATH = System.getenv("PATH").split(File.pathSeparator);
```

Reply:
> This is probably out of scope therefore feel free to ignore this comment.

I agree. I don't want to dive in this motification in this PR. Thanks for the comment.